go/api/commands.go

```diff
@@ -371,4 +371,18 @@ type StringCommands interface {
 	//
 	//[valkey.io]: https://valkey.io/commands/getdel/
 	GetDel(key string) (string, error)
+
+	// Del removes the specified keys. A key is ignored if it does not exist.
+	//
+	// Parameters:
+	//  keys - One or more keys to delete.
+	//
+	// Return value:
+	//  Returns the number of keys that were removed.
+	//
+	// For example:
+	// 	result, err := client.Del("key1", "key2", "key3")
```

Comment:
Please add how the result will look like. (You can refer the templates of other commands)